astrowidgets/tests/widget_api_test.py

```diff
@@ -243,10 +243,48 @@ def test_add_markers(self):
         with pytest.raises(ValueError, match="No markers named 'test2'"):
             self.image.get_markers(marker_name='test2')
 
+    def test_get_markers_accepts_list_of_names(self):
+        # Check that the get_markers method accepts a list of marker names
+        # and returns a table with all the markers from all the named sets.
+        rng = np.random.default_rng(1234)
+        data = rng.integers(0, 100, (5, 2))
```

Comment:
Is randomness really necessary here?

Reply:
No, that is just what GitHub Copilot inserted 😬